docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md

```diff
@@ -0,0 +1,80 @@
+        //test actions
+    })
+```
+
+You can use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout) as well.
+
+```js
+fixture`Setting Timeouts`
+    .page`http://devexpress.github.io/testcafe/example`;
+
+test
+    .timeouts({
+        pageLoadTimeout:    2000,
+        pageRequestTimeout: 60000,
+        ajaxRequestTimeout: 60000
+    })
+    ('My test', async t => {
+        //test actions
+    })
+```
+
+## Bug Fixes
+
+* Fixed a bug where TestCafe would sometimes be unable to trigger a `hover` event on a `radio` element ([#5916](https://github.com/DevExpress/testcafe/issues/5916))
+* Fixed a bug where TestCafe was unable to register a Service Worker due to the wrong `currentScope` calculation inside a `Window.postMessage` call ([testcafe-hammerhead/#2524](https://github.com/DevExpress/testcafe-hammerhead/issues/2524))
+* `RequestLogger` now shows a correct protocol for WebSocket requests ([testcafe-hammerhead/#2591](https://github.com/DevExpress/testcafe-hammerhead/issues/2591))
+* Test execution now pauses when the browser window is in the background ([testcafe-browser-tools/#158](https://github.com/DevExpress/testcafe-browser-tools/issues/158))
+* TestCafe now appends an extension to screenshot filenames ([#5103](https://github.com/DevExpress/testcafe/issues/5103))
+* Fixed a bug where TestCafe would emit test action events after the end of a test run ([#5650](https://github.com/DevExpress/testcafe/issues/5650))
+* TestCafe now closes if the `No tests to run` error occurs in Live mode ([#4257](https://github.com/DevExpress/testcafe/issues/4257))
+* Fixed a freeze that happened when you run a test suite with skipped tests ([#4967](https://github.com/DevExpress/testcafe/issues/4967))
+* Fixed an error where a `documentElement.transform.translate` call moved the TestCafe UI in the browser window ([#5606](https://github.com/DevExpress/testcafe/issues/5606))
+* TestCafe now emits a warning if you pass an unawaited selector to an assertion ([#5554](https://github.com/DevExpress/testcafe/issues/5554))
+* Fixed a crash that sometimes occurred in Chrome v85 and earlier on pages with scripts ([PR testcafe-hammerhead/#2590](https://github.com/DevExpress/testcafe-hammerhead/pull/2590))
+* Overrode the [insertAdjacentElement method](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement) inside TestCafe ([testcafe-hammerhead/#1321](https://github.com/DevExpress/testcafe-hammerhead/issues/1321))
+* Overrode the internal `Element.prototype.after` method ([testcafe-hammerhead/#1833](https://github.com/DevExpress/testcafe-hammerhead/issues/1833))
+* Overrode the [ParentNode.prepend method](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/prepend) inside TestCafe ([testcafe-hammerhead/#1005](https://github.com/DevExpress/testcafe-hammerhead/issues/1005))
```

Comment:
Are these three fixes relevant to users?

Reply:
I think we can safely omit those